src/dune_rules/coq_rules.mli

```diff
@@ -35,3 +45,24 @@ val extraction_rules :
   -> dir_contents:Dir_contents.t
   -> Extraction.t
   -> Action.Full.t Action_builder.With_targets.t list Memo.Build.t
+
+val deps_of :
```

Comment:
I suggest we add a brief docstring if you have the energy.

Reply:
Also done.